jenkins/revert-master.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -o errexit
+read -p "Delete master [press enter]"
+gcloud compute instances delete pull-jenkins-master
+read -p "Delete master disks [press enter]"
+gcloud compute disks delete pull-jenkins-master pull-jenkins-master-data pull-jenkins-master-docker
```

Comment:
Sounds like a great improvement if we actually want to invest further in this script. Right now I just want something in source control.